modules/ocf_printhost/manifests/cups.pp

```diff
@@ -0,0 +1,48 @@
+class ocf_printhost::cups {
+  package { [ 'cups', 'cups-bsd' ]: }
+  ocf::repackage { 'hplip':
+    recommends => false
+  }
+  service { 'cups': }
+
+  File {
+    require => Package['cups'],
+    notify  => Service['cups'],
+  }
+
+  file {
+    '/etc/cups/cupsd.conf':
+      content => template('ocf_printhost/cups/cupsd.conf.erb');
+    '/etc/cups/cups-files.conf':
+      content => template('ocf_printhost/cups/cups-files.conf.erb');
+    '/etc/cups/lpoptions':
+      content => 'Default double';
+    ['/etc/cups/raw.convs', '/etc/cups/raw.types']:
+      content => '# deny printing raw jobs';
```

Comment:
text files should end with a single newline:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

(one of my personal pet-peeves. not a huge deal here as long as the tool reads it properly, but especially annoying when people commit actual source code without a newline)